pages/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "@exadel/esl-website",
+  "version": "1.0.0",
+  "private": true,
+  "description": "ESL Website source",
+  "homepage": "https://esl-ui.com/",
+  "license": "ISC",
+  "sideEffects": [
+    "pages/src/**/*.ts"
+  ],
+  "scripts": {
+    "start": "npm run clean && concurrently --kill-others \"npm run watch:lib\" \"npm run watch:ts\" \"npm run watch:less\" \"npm run watch:eleventy\"",
+    "clean": "rimraf \"dist\"",
+    "watch:lib": "cd .. && npm run watch",
+    "watch:ts": "webpack --watch",
+    "watch:less": "chokidar --initial \"src/**/*.less\" -c \"npm run build-pages:less\"",
+    "watch:eleventy": "eleventy --serve --incremental -- --env=development",
+    "minify:css": "cleancss -o \"dist/bundles/localdev.css\" \"dist/bundles/localdev.css\"",
+    "build-pages": "npm run clean && concurrently \"npm run build-pages:ts\" \"npm run build-pages:less && npm run minify:css\" && npm run build-pages:eleventy",
+    "build-pages:ts": "webpack --mode=production",
+    "build-pages:less": "lessc \"src/localdev.less\" \"dist/bundles/localdev.css\" --source-map=dist/bundles/localdev.css.map",
+    "build-pages:eleventy": "eleventy -- --env=production",
+    "build-pages:analyzer": "webpack --mode=production --profile --json ../.report/stats.json && webpack-bundle-analyzer ../.report/stats.json dist/bundles -s gzip -m static -r ../.report/stats.html"
```

It is possible to use shell variables to store the file path, but not sure that it will work under Windows. Needs checking.
```suggestion
    "build-pages:analyzer": "STATS=../.report/stats webpack --mode=production --profile --json $STATS.json && webpack-bundle-analyzer $STATS.json dist/bundles -s gzip -m static -r $STATS.html"
```

---

Vote for moving it to js at all )

I will create a point in follow-up for that case anyway and will mention both suggested options.